src/core/channelz/channelz.h

```diff
@@ -49,7 +49,12 @@
 #include "src/core/util/useful.h"
 
 // Channel arg key for channelz node.
-#define GRPC_ARG_CHANNELZ_CHANNEL_NODE "grpc.internal.channelz_channel_node"
+#define GRPC_ARG_CHANNELZ_CHANNEL_NODE \
+  "grpc.internal.no_subchannel.channelz_channel_node"
```

Comment:
Please use the existing GRPC_ARG_NO_SUBCHANNEL_PREFIX macro for these two channel args, to make sure the strings stay in sync.